vm/systemSmartContracts/auction.go

```diff
@@ -1123,6 +1165,12 @@ func (s *stakingAuctionSC) deleteUnBondedKeys(registrationData *AuctionData, unB
 }
 
 func (s *stakingAuctionSC) claim(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if s.flagStakingV2.IsSet() {
+		//claim function will become unavailable after enabling staking v2
+		s.eei.AddReturnMessage("invalid method to call")
```

Comment:
change return message to "claim function is disabled"

Reply:
will do

Reply:
done